### PR TITLE
CC-23635 Introduced cart prices feature IG.

### PR DESCRIPTION
**Pull request by romanhavrylko**
## PR Description

Ticket: https://spryker.atlassian.net/browse/CC-26296

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+```php
+<?php
+
+namespace Pyz\Zed\PriceCartConnector;
+
+use Spryker\Zed\PriceCartConnector\PriceCartConnectorConfig as SprykerPriceCartConnectorConfig;
+
+class PriceCartConnectorConfig extends SprykerPriceCartConnectorConfig
+{
+    protected const IS_ZERO_PRICE_ENABLED_FOR_CART_ACTIONS = false;
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md:53]: please add docblock

File: _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+}
+```
+
+If `IS_ZERO_PRICE_ENABLED_FOR_CART_ACTIONS=false`, while attempting to add the product with zero price to the cart, you get the following message: "Price in selected currency not found for product with sku '%sku%'. Please change the currency or remove product from order.".
+
+The `PriceCartConnectorConfig::getItemFieldsForIdentifier()` allows to set up a list of fields that are used to build items identifiers. Based on generated identifiers, system can recognize duplicate items and perform requests only for unique items.
+
+{% info_block warningBox "Warning" %}
+
+Apply the following changes only if you have the [Marketplace Product Offer Prices](/docs/marketplace/dev/feature-integration-guides/{{site.version}}/marketplace-product-offer-prices-feature-integration.html) feature installed.
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md:75]: this section looks like a part from another IG

File: _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+{
+	/**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Spryker\Zed\CartExtension\Dependency\Plugin\ItemExpanderPluginInterface>
+     */
+    protected function getExpanderPlugins(Container $container): array
+	{
+		return [
+            new CartItemPricePlugin(),
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md:169]: formatting is broken here

Comment (DmitryLymarenko) [reply _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md:169]: same for similar rows below

File: _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md (hunk 2 above)
Comment (romanhavrylko) [reply _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md:78]: Discussed and decided to leave it as it is.

**Review by DmitryLymarenko — APPROVED** [review]

**Review by philinecao — APPROVED** [review]

File: _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+{% endinfo_block %}
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME                                        | VERSION          | INTEGRATE GUIDE                                                                                                                                                 |
+|---------------------------------------------|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Cart                                        | {{site.version}} | [Cart feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/cart-feature-integration.html)                                            |
+| Prices                                      | {{site.version}} | [Prices feature integration](/docs/pbc/all/price-management/{{site.version}}/install-and-upgrade/install-features/install-the-prices-feature.html)              |
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md:19]:
```suggestion
| Prices                                      | {{site.version}} | [Install the Prices feature](/docs/pbc/all/price-management/{{site.version}}/install-and-upgrade/install-features/install-the-prices-feature.html)              |
```

File: _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+| Marketplace Product Offer Prices (Optional) | {{site.version}} | [Marketplace Product Offer Prices](/docs/marketplace/dev/feature-integration-guides/{{site.version}}/marketplace-product-offer-prices-feature-integration.html) |
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md:20]:
```suggestion
| Marketplace Product Offer Prices (Optional) | {{site.version}} | [Marketplace Product Offer Prices feature integration](/docs/marketplace/dev/feature-integration-guides/{{site.version}}/marketplace-product-offer-prices-feature-integration.html) |
```

File: _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md [hunk 6]
```diff
@@ -0,0 +1,208 @@
+
+
+
+## Install feature core
+
+{% info_block errorBox %}
+
+The following feature integration guide expects the basic feature to be in place. The current feature integration guide only adds the **Cart Prices functionality**.
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md:8]:
```suggestion
The following feature integration guide expects the basic feature to be in place. The current feature integration guide only adds the *Cart Prices functionality*.
```

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**